tests/conftest.py

```
@@ -8,34 +8,72 @@
+            os.environ[var] = default
 
     # Get the project root directory (parent of tests directory)
     current_dir = os.path.dirname(os.path.abspath(__file__))
     project_root = os.path.dirname(current_dir)
+    
+    # Use a different port for tests to avoid conflicts
+    test_port = 5001
 
-    # Start the FastAPI app using pipenv
+    # Start the FastAPI app using pipenv, with output visible for debugging
     process = subprocess.Popen([
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)